### ISSUE TITLE
False Positive on html2pdf

### ISSUE THREAD

**Issue by snemets** [dependency-check/DependencyCheck#4021]
False positive on library html2pdf.jar - reported as `cpe:2.3:a:html2pdf_project:html2pdf:*:*:*:*:*:*:*:*`
```xml
<dependency>
    <groupId>com.itextpdf</groupId>
    <artifactId>html2pdf</artifactId>
    <version>4.0.0</version>
</dependency>
```

**Comment by freddiN** [comment]
This is mixing up two different html2pdf projects:

html2pdf-3.0.5.jar (pkg:maven/com.itextpdf/html2pdf@3.0.5, cpe:2.3:a:html2pdf_project:html2pdf:3.0.5:*:*:*:*:*:*:*) : CVE-2021-45394

We use this one:
https://github.com/itext/itext7
```
<groupId>com.itextpdf</groupId>
<artifactId>html2pdf</artifactId>
```

The one affected by CVE 2021-45394:
https://github.com/spipu/html2pdf/

**Comment by snemets** [comment]
Exactly. The  CVE 2021-45394 vulnerability should be connected to the spipu/html2pdf project, but OWASP marks a com.itextpdf/html2pdf as vulnerable also.

**Comment by freddiN** [comment]
could this get some attention?
I see many false positives fixed with 7.0.0 and 7.0.1, but this one is still in there. 

UPDATE: Ahh I think it was the wrong format, there is a special kind of Issue for a "False Positive Report", it seems

**Comment by freddiN** [comment]
I resubmitted it using the "False Positive Report" template, feel free to close both issues once this is resolved.
